com.unity.sg2/Editor/GraphUI/EditorCommon/CommandStateObserver/ShaderGraphCommandsRegistrar.cs

```diff
@@ -112,6 +56,40 @@ public static class ShaderGraphCommandsRegistrar
                 graphViewModel.GraphModelState,
                 previewManager);
 
+
+            // Demo commands (TODO: Remove)
+            //commandDispatcher.RegisterCommandHandler<UndoStateComponent, GraphViewStateComponent, AddPortCommand>(
+            //    AddPortCommand.DefaultCommandHandler,
+            //    graphTool.UndoStateComponent,
+            //    graphView.GraphViewState
+            //);
+
+            //commandDispatcher.RegisterCommandHandler<UndoStateComponent, GraphModelStateComponent, RemovePortCommand>(
+            //    RemovePortCommand.DefaultCommandHandler,
+            //    graphTool.UndoStateComponent,
+            //    graphView.GraphModelState
+            //);
```

Comment:
Either delete commented code or put a longer explanation in for why we should keep it.

Reply:
oh yeah I deleted those files and forgot to remove these, thanks for calling it out